### DISCUSSION TITLE
Cannot Sign in with Apple ID

### DISCUSSION THREAD

**Discussion by TheCarpetMerchant** [codemagic-ci-cd/codemagic-docs#1930] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

XCode -> Settings -> Accounts -> + -> Apple ID

### Expected results

Signing in with an Apple ID

### Actual results

"The action could not be performed"

**Comment by kulakarla** [comment]
@TheCarpetMerchant 

Hi, the question is a bit unclear - are you trying to log in to Xcode on your local mac machine and you're getting the error there?

**Comment by TheCarpetMerchant** [comment]
No, I'm trying to log in on one of your VMs.

**Reply by kulakarla** [reply, marked as answer]
What are you trying to achieve, are you trying to sign your app in our VM-s manually? Signing in to Apple ID from Virtual Machines is impossible, see [here](https://developer.apple.com/forums/thread/707682).
